src/core/regimes.rkt

```diff
@@ -121,8 +121,9 @@
 
   (define bit-err-lsts* (map (curry map ulps->bits) err-lsts*))
 
-  (define can-split? (append (list #f)
-                             (for/list ([val (cdr splitvals*)] [prev splitvals*])
+  (define can-split? (vector-append (vector #f)
```

Comment:
So this is fine, but I wonder if we can skip the `vector-append` and just make the `can-split?` vector one element shorter than the other vectors. I think the only change would be doing `(in-vector can-split)` instead of `(in-vector can-split 1)`.

Reply:
Ok, we'd need to adjust a few other accesses to `can-split`.